service/history/ndc/workflow_resetter.go

```diff
@@ -32,6 +32,8 @@ import (
 	"fmt"
 	"time"
 
+	"google.golang.org/protobuf/types/known/timestamppb"
```

Comment:
Make sure you don't add newlines to imports, we discussed this as a team and decided not to separate temporal imports from other imports. You probably have some configuration in your IDE that automatically does this.